src/components/Trade/TradeInputV2.tsx

```diff
@@ -302,6 +302,16 @@ export const TradeInput = () => {
     }
   }, [getTranslationKey])
 
+  const sellAmountTooSmall = useMemo(() => {
```

Comment:
This will fail in case of "Not enough ETH to cover gas", unfortunately, not enough ETH to cover gas and too small amount to sell aren't mutually exclusive:
- users could try to enter a small amount without enough ETH to cover gas (as is the case in this screenshot)
- users could not have enough ETH to cover gas, but enter a high enough sell amount

As a result, we can't just add it here and have to find better heuristics that work in both cases

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/17035424/193805294-c3d0e983-0c77-4f89-9330-01601ef2d616.png">

Reply:
856760792f1e5b950a6ca1d56b94d4d05d772d90